src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -625,6 +674,24 @@ public void finalizeOutputFile(int testStatus) {
         if (System.getProperty("packageResults") != null && "true".equals(System.getProperty("packageResults"))) {
             packageTestResults();
         }
+        if (System.getProperty("generatePDF") != null && "true".equals(System.getProperty("generatePDF"))) {
+            generatePdf();
+        }
+    }
+
+    /**
+     * Generates a pdf report in the same directory as the html report
+     */
+    private void generatePdf() {
+        String pdfFilename = directory + "/" + filename + ".pdf";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1075)